lib/components/reports/reportsGraph.dart

```diff
@@ -4,13 +4,25 @@ import 'package:flutter/rendering.dart';
 import 'package:off_top_mobile/components/subnavbar.dart';
 import 'package:charts_flutter/flutter.dart' as charts;
 import 'package:off_top_mobile/components/reports/scores.dart';
+import 'package:off_top_mobile/components/reports/getTableDataWidget.dart';
+import 'package:off_top_mobile/models/userSession.dart';
+import 'package:off_top_mobile/userReportTable.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
 
 class ReportsGraph extends StatefulWidget {
+  const ReportsGraph(this.date);
+  final String date;
   @override
   _ReportsState createState() => _ReportsState();
 }
 
+
+
+
 class _ReportsState extends State<ReportsGraph> {
+  String date;
+
   List<charts.Series<Scores, int>> _seriesLineData;
   void _generateData() {
     var linesscoresdata1 = [
```

Comment:
instead of var explicitly write out the data type (List<Scores>) work out in place of var